src/HostRouter.ts

```diff
@@ -24,14 +24,20 @@ export class HostRouter {
    *
    * @param route The route to lookup, such as '/foo/bar/baz'
    */
-  public getClientTarget(route: string): ClientTarget {
+  public getClientTarget(route: string): ClientTarget | null {
+    const desiredRoute = String(route || '');
```

Comment:
I don't think you need the string constructor here? `route || ''` should suffice.